### DISCUSSION TITLE
建议增加以下功能

### DISCUSSION THREAD

**Discussion by zzr-shift** [Meloong-Git/PCL#4600] [提问] [answered]
1.在已安装的版本的版本设置中提供Mod加载器下载，并在下载完成后自动添加到游戏所在文件夹并对所显示的版本名称进行更改为“原名+mod加载器版本”
2.可以自行选择启动器字体（思源黑体重度患者）
3.增加光影下载

**Comment by 3gf8jv4dv** [comment, marked as answer]
> 1.在已安装的版本的版本设置中提供Mod加载器下载，并在下载完成后自动添加到游戏所在文件夹并对所显示的版本名称进行更改为“原名+mod加载器版本”

#220

> 2.可以自行选择启动器字体（思源黑体重度患者）

#366

> 3.增加光影下载

#396, #4359

---

It is recommended to check the issues list and #1930 first.
建议先看一遍 issues 列表和 #1930。
